src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { Modal } from 'react-bootstrap';
-import browser from 'detect-browser';
+const browser = require('bowser');
```

Comment:
@inodb @alisman Sorry about that. At first, I only read the first part of your comment Ino so I only removed `detect-browser`. I had forgotten to add `bowser`.  Now it should be fixed.

Also, the tests only tests the `handleUnsupportedBrowser` fn.  They do not use `detect-browser` or `bowser`.  If you remember, when I first made this component about 2 months ago, we could not get the tests to work properly with the package.